cfme/tests/intelligence/reports/test_reports_schedules.py

```diff
@@ -246,3 +251,50 @@ def test_reports_schedules_user(appliance, request, user, schedule_data):
     assert schedule.exists
     view = schedule.create_view(ScheduleDetailsView)
     assert view.schedule_info.get_text_of("To E-mail") == f"{user.name} ({user.email})"
+
+
+@pytest.mark.customer_scenario
+@pytest.mark.meta(automates=[1740229])
+def test_miq_schedule_validation_failed(temp_appliance_preconfig):
+    """
+    Bugzilla:
+        1729441
+        1740229
+
+    Polarion:
+        assignee: pvala
+        casecomponent: Reporting
+        initialEstimate: 1/10h
+        testSteps:
+            1. Restore the customer database and monitor the EVM logs while restoring.
+            2. Restart the server.
+        expectedResults:
+            1. Should not encounter the following message(refer BZ description) in the logs.
+            2. Server should restart successfully.
+    """
+    appliance = temp_appliance_preconfig
+    dump = FTPClientWrapper(cfme_data.ftpserver.entities.databases).get_file("miqschedule_dump")
+    dump_destination = os.path.join("/tmp", dump.name)
+
+    # Download the customer db on appliance
+    if not appliance.ssh_client.run_command(
+        f"curl -o {dump_destination} ftp://{dump.link}"
+    ).success:
+        pytest.fail("Failed to download the file to the appliance.")
+
+    # In case the failure pattern is encountered, the following piece of code will raise
+    # a TimedOutError from appliance.wait_for_web_ui
+    # The matched_patterns will only be found on successful database restore.
+    with LogValidator(
+        "/var/www/miq/vmdb/log/evm.log",
+        failure_patterns=[
+            ".* ERROR .*Validation failed: MiqSchedule: Name has already been taken  Method.*"
+        ],
```

Comment:
Are you sure? we have extra space here: taken  Method.

Reply:
Yes, it is correct.

Reply:
Do we need to search for the whole string? Can we just search for the `.* ERROR .*Validation failed: MiqSchedule:`. My concern is that this could change upstream and then we may be missing test failures.

Reply:
Um, I'm not sure I completely understand your concern. Have you faced something like this before where a log message changed in upstream?

Reply:
I haven't, probably not really something worth fretting over. But that log message looks like a typo upstream and it could be fixed at some point.

Reply:
Oh I see, is it because of that extra space? Um, I guess I can make some change there.

Reply:
So I added a few `.*` :grimacing: